java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_2/Tree.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class Tree {
+
+    class Node {
```

Comment:
does this class needs to have a `default` visibility level? do we plan to expose it to some code outside of this class?